libsel4/include/sel4/macros.h

```diff
@@ -37,6 +37,12 @@
 #define LIBSEL4_INLINE __attribute__((noinline)) __attribute__((unused)) __attribute__((weak))
 #endif
 
+#ifdef CONFIG_LIB_SEL4_PUBLIC_SYMBOLS
+#undef LIBSEL4_INLINE
+#define LIBSEL4_INLINE __attribute__((unused)) __attribute__((weak))
+#define LIBSEL4_INLINE_FUNC __attribute__((unused)) __attribute__((weak))
```

Comment:
Should there be an `#else` branch defining `LIBSEL4_INLINE_FUNC` back to `static inline`?